app/views/templates/_home.html.erb

```diff
@@ -16,61 +18,78 @@
       </p>
     </div>
     
-    <div class="small-12 medium-6 columns">
+    <div class="small-12 medium-5 columns text--fira-sans text--18 text--grey-dark">
       <p>
         <strong>Biodiversity and ecosystem services are essential components of a healthy economy,</strong> relvant across all sectors of society. However, the continued loss of the Worlds biodiversity is having significant consequences across a range of sectors including poverty reduction and development.
       </p>
+      
+      <div class="icon-wrapper">
+        <%= link_to '/', title: '', class: 'icon--flag-link' do %>
```

Comment:
I believe we can do `root_path` instead of `/` and it will always redirect to the homepage. We can remove the title attribute if it's not being used too